README.md

```diff
@@ -61,35 +44,27 @@ export SECRET_KEY=xxxxx
 export PROJECT_ID=1a5
 ```
 
-2- Edit the tox.ini file in v2_validation directory `tests/v2_validation/tox.ini` to run the specific tests you need and make sure to add `passenv=*`, it should look something like that:
+2 - Edit the tox.ini file in v2_validation directory `tests/v2_validation/tox.ini` to run the specific tests if needed, change by adding to the command:
```

Comment:
You don't need to edit the tox.ini to pass the OS ENV variables anymore (that is what passenv=* does), but if you want to run a subset of tests instead of the whole suite, this script does not do this yet. This part of the readme details what is still required to change in the tox.ini if you want to run a subset of tests.